api/v1/strict_server.go

```diff
@@ -248,6 +250,12 @@ func (srv *StrictServerImpl) GetRuntimeTransactions(ctx context.Context, request
 		if err2 != nil {
 			return nil, fmt.Errorf("round %d tx %d: %w", storageTransaction.Round, storageTransaction.Index, err2)
 		}
+		// address preimage may not be available
```

Comment:
Alternatively move this logic into `api/v1/logic.go`?

Unfortunately, we don't get the Oasis address until post raw body parsing of the transaction, so we have to do a second loop for look up 🤢 🤔

Reply:
Thanks! Rabbit hole took me to `runtime_transaction_signers` eventually.

Reply:
signers is only a subset though